### PR TITLE
[CRW 2.2] Update Go devfile with new sample

### PR DESCRIPTION
**Pull request by svor**
Signed-off-by: svor <vsvydenk@redhat.com>

### What does this PR do?
Use [Golang Health Check sample](https://github.com/crw-samples/golang-health-check.git) as a new sample for Go devfile
screenshot-nimbus-capture-2020 03 30-19_10_35
screenshot-nimbus-capture-2020 03 30-19_11_16

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-740

### CODE REVIEW

File: dependencies/che-devfile-registry/devfiles/05_go/devfile.yaml [hunk 1]
```diff
@@ -77,6 +64,8 @@ commands:
             "request": "launch",
             "mode": "auto",
             "program": "${fileDirname}",
-          },
+            "env": {},
```
Comment (nickboldt) [inline dependencies/che-devfile-registry/devfiles/05_go/devfile.yaml:67]: do we need these empties?

**Comment by nickboldt** [comment]
please resolve conflicts

File: dependencies/che-devfile-registry/devfiles/05_go/devfile.yaml (hunk 1 above)
Comment (svor) [reply dependencies/che-devfile-registry/devfiles/05_go/devfile.yaml]: no, we don't, it'll work without them as well

**Comment by svor** [comment]
@nickboldt are we OK to merge [CRW 2.2]... PRs?

**Comment by nickboldt** [comment]
Yeah, I guess it's time. :D feel free to merge'em all!

**Pull request merged**